avivator/src/components/Controller/components/ChannelController.jsx

```diff
@@ -61,16 +65,37 @@ function ChannelController({
     shallow
   );
   const rgbColor = toRgb(colormap, color);
-  const [min, max] = domain;
-  // If the min/max range is and the dtype is float, make the step size smaller so contrastLimits are smoother.
+  const domainReducer = useCallback(
+    (state, action) => {
+      switch (action) {
+        case 'max/min': {
+          return domain;
+        }
+        case 'full': {
+          const { dtype } = loader[0];
+          const { max } = DTYPE_VALUES[dtype];
+          // Min is 0 for unsigned, or the negative of the max for signed dtypes.
+          const min =
+            dtype.startsWith('Int') || dtype.startsWith('Float') ? -max : 0;
```

Comment:
When would there be negative values?

Reply:
Signed data types?

Reply:
Also, floats can technically be negative, no?  Unless I am misremembering.

Reply:
Well they can but I am not sure if it's used this way in the microscopy community.

Reply:
Oh cool, officially from OME it's non-negative so will fix this: https://downloads.openmicroscopy.org/bio-formats/5.1.10/api/ome/xml/model/primitives/package-summary.html